x-pack/plugins/siem/public/components/tables/helpers.tsx

```diff
@@ -173,3 +173,24 @@ export const getRowItemOverflow = (
     </>
   );
 };
+
+export const OverflowField = React.memo<{
+  value: string;
+  showToolTip?: boolean;
+  overflowLength?: number;
+}>(({ value, showToolTip = true, overflowLength = 50 }) => (
+  <span>
+    {showToolTip ? (
+      <EuiToolTip data-test-subj={'message-tooltip'} content={'message'}>
+        <>{value.substring(0, overflowLength)}</>
+      </EuiToolTip>
+    ) : (
+      <>{value.substring(0, overflowLength)}</>
+    )}
+    {value.length > overflowLength && (
+      <EuiToolTip content={value}>
+        <MoreRowItems type="boxesHorizontal" />
+      </EuiToolTip>
+    )}
+  </span>
```

Comment:
Do you need `<span>` or could you use a Fragment? `<>`?

Reply:
The `<MoreRowItems>` will wrap to the next line without the `<span>`.